### PR TITLE
[16.0][ADD] pos_disable_product_attribute module

### PR DESCRIPTION
**Pull request by ecino**
Module for hiding specific product attibutes in POS

### CODE REVIEW

**Comment by ecino** [comment]
Can someone review this please? Thanks!

**Comment by ecino** [comment]
@OCA/pos-maintainers Anyone willing to review?

File: pos_disable_product_attribute/models/product_attribute.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# @author Emanuel Cino
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo import api, fields, models
+
+
+class ProductAttribute(models.Model):
+    _inherit = "product.attribute"
+
+    pos_visibility = fields.Selection(
```
Comment (ivantodorovich) [inline pos_disable_product_attribute/models/product_attribute.py:11]: Not-blocking: I'd align to the standard `available_in_pos` in products and make this a Boolean, with the same name

But perhaps it's only my opinion here, feel free to ignore

**Review by ivantodorovich — APPROVED** [review]

**Comment by ecino** [comment]
It's one point of view that is valid. Mine was to do the same as another visibility field already present in this model for the website : https://github.com/odoo/odoo/blob/d63d26d8e766ebec1d3efa705a5b6d6a12b0a1d0/addons/website_sale/models/product_attribute.py#L12

**Pull request closed without merging**